presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -715,6 +715,59 @@ public void testGetColumns()
                 assertFalse(rs.next());
             }
         }
+
+        try (Connection connection = createConnection("blackhole", "blackhole");
+            Statement statement = connection.createStatement()) {
+                assertEquals(statement.executeUpdate(
+                    "CREATE TABLE test_get_columns_table (" +
+                        "c1 boolean, " +
+                        "c2 bigint, " +
+                        "c3 integer, " +
+                        "c4 smallint, " +
+                        "c5 tinyint, " +
+                        "c6 double, " +
+                        "c7 varchar(1234), " +
+                        "c8 varbinary, " +
+                        "c9 time, " +
+                        "c10 \"time with time zone\", " +
+                        "c11 timestamp, " +
+                        "c12 \"timestamp with time zone\", " +
+                        "c13 date, " +
+                        "c14 decimal(8,2), " +
+                        "c15 decimal(38,0)" +
+                        ")"), 0);
+        }
+
+        try (Connection connection = createConnection()) {
+            try (ResultSet rs = connection.getMetaData().getColumns("blackhole", "blackhole", "test_get_columns_table", null)) {
+                assertColumnMetadata(rs);
+                assertColumnSpec(rs, null,          null,   0L);    // boolean
```

Comment:
These aligned columns look nice, but will be messed up when someone runs the code formatter. Please format with one space after arg. It's fine to keep the comments aligned.

https://github.com/airlift/codestyle